src/html/authorize.html.template

```diff
@@ -1,69 +1,96 @@
-          <path d="M1615.01 272.606C1614.16 268.914 1613.6 265.158 1613.35 261.376C1610.28 265.78 1606.08 269.273 1601.19 271.486C1595.48 274.09 1589.26 275.375 1582.98 275.246C1572.17 275.246 1563.5 272.366 1556.98 266.606C1550.46 260.846 1547.21 253.05 1547.22 243.216C1547.22 233.576 1550.43 226.04 1556.84 220.606C1563.25 215.173 1572.02 212.453 1583.15 212.446H1612.8V208.646C1612.8 202.46 1610.99 197.836 1607.38 194.776C1603.77 191.716 1598.71 190.186 1592.2 190.186C1580.68 190.186 1572.41 195.003 1567.39 204.636L1550.21 194.576C1558.54 178.89 1572.96 171.046 1593.47 171.046C1605.9 171.046 1616.08 174.236 1624.03 180.616C1631.98 186.996 1635.98 197.38 1636.03 211.766V251.026C1636.03 262.153 1636.61 269.346 1637.78 272.606H1615.01ZM1613.01 228.756H1588.69C1583.16 228.756 1578.87 230.026 1575.8 232.566C1574.28 233.837 1573.08 235.441 1572.28 237.254C1571.48 239.066 1571.12 241.039 1571.21 243.016C1571.21 247.636 1572.77 251.153 1575.9 253.566C1579.03 255.976 1583.48 257.176 1589.28 257.176C1596.24 257.176 1601.94 254.98 1606.37 250.586C1610.8 246.193 1613.02 239.666 1613.01 231.006V228.756Z" class="wordmark"/>
-          <path d="M1749.53 211.766V272.606H1726V216.826C1726 207.446 1723.79 201.033 1719.36 197.586C1715.18 194.155 1709.92 192.323 1704.51 192.416C1701.37 192.323 1698.24 192.853 1695.31 193.975C1692.38 195.098 1689.69 196.791 1687.42 198.956C1682.87 203.316 1680.6 209.533 1680.59 217.606V272.606H1656.96V173.976H1679.12L1679.9 184.616C1682.96 180.236 1687.19 176.803 1692.11 174.706C1697.6 172.234 1703.57 170.986 1709.59 171.046C1722.35 171.046 1732.2 174.576 1739.13 181.636C1746.06 188.696 1749.53 198.74 1749.53 211.766Z" class="wordmark"/>
-          <path d="M1824.98 272.606C1819.35 273.952 1813.59 274.696 1807.8 274.826C1797.84 274.826 1790.08 272.09 1784.51 266.616C1778.94 261.143 1776.16 253.006 1776.16 242.206V193.996H1761.02V173.996H1776.16V143.216H1799.59V173.976H1822.35V193.976H1799.59V237.976C1799.59 248.643 1803.73 253.976 1812 253.976C1815.42 253.833 1818.8 253.14 1822 251.926L1824.98 272.606Z" class="wordmark"/>
-        </svg>
-      </div>
-      <ha-authorize></ha-authorize>
-    </div>
+    <ha-authorize></ha-authorize>
     <%= renderTemplate("_js_base.html.template") %>
     <%= renderTemplate("_preload_roboto.html.template") %>
     <script crossorigin="use-credentials">
       // Safari 12 and below does not have a compliant ES2015 implementation of template literals, so we ship ES5
-      if (!isS11_12) {
+      window.latestJS = window.latestJS || !isS11_12;
+      if (window.latestJS) {
         <% for (const entry of latestEntryJS) { %>
           import("<%= entry %>");
         <% } %>
-        window.latestJS = true;
```

Comment:
We are testing here if `import()` will work, if it throws it doesnt get here so `latestJS` is not set to true.

With your code `latestJS` is always set to `true` except when the browser is Safari 11/12?